packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/imageEditors/Rotator.ts

```diff
@@ -51,19 +51,22 @@ export function updateRotateHandlePosition(
     rotateCenter: HTMLElement,
     rotateHandle: HTMLElement
 ) {
-    const top = rotateHandle.getBoundingClientRect()?.top - editorRect?.top;
-    const { angleRad, heightPx } = editInfo;
-    const cosAngle = Math.cos(angleRad);
-    const adjustedDistance =
-        cosAngle <= 0
-            ? Number.MAX_SAFE_INTEGER
-            : (top + heightPx / 2 + marginVertical) / cosAngle - heightPx / 2;
+    const rotateHandleRect = rotateHandle.getBoundingClientRect();
```

Comment:
Can we also add a null check in rotate handle, before calling getBoundingClientRect? I think the callstack is saying that rotateHandle could be null
```
Uncaught TypeError: Cannot read properties of undefined (reading 'getBoundingClientRect')